packages/web-service/src/pages/habitat/a24/sett/router.js

```diff
@@ -0,0 +1,22 @@
+import settStart from './sett-start.js'
```

Comment:
Can be discussed to pull the routes into simpler manageable files. (Will help with any common work in the future)